runtime/syntax/vim.vim

```diff
@@ -644,7 +651,7 @@ syn region	vimGlobal	matchgroup=Statement start='\<v\%[global]!\=/' skip='\\.' e
 "   g:vimsyn_embed =~# 'r' : embed ruby
 "   g:vimsyn_embed =~# 't' : embed tcl
 if !exists("g:vimsyn_embed")
- let g:vimsyn_embed = 'l'
+ let g:vimsyn_embed= "lmpPr"
```

Comment:
Restricting this to Lua embedding may also have been intentional.

Reply:
I think there was a perf performance but lua being first class citizen was not axed.